ui/pages/keychains/index.scss

```diff
@@ -56,23 +71,6 @@
   }
 }
 
-.import-account {
```

Comment:
`.import-account` is already specified above, so this code was merged with that above.